libr/util/print.c

```diff
@@ -701,6 +771,74 @@ R_API int r_print_string(RPrint *p, ut64 seek, const ut8 *buf, int len, int opti
 	return i;
 }
 
+R_API int r_print_gameboy_string(RPrint *p, ut64 seek, const ut8 *buf, int len, int options) {
```

Comment:
this function should be encode/decode from/to utf8/ascii. this way we can reuse the rest of functions and commands in r2 to process the given string